app/interactors/images/create.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+class Images::Create
```

Comment:
That feels pretty weird because of the class name being a verb, if that seems cool to you and others though can roll with that - is consistent at least.